### ISSUE TITLE
Fix the home page accessibility

### ISSUE THREAD

**Issue by duffn** [duffn/theouterrim#154]
Updating the home page in #147 made the Lighthouse accessibility score plummet by 12 points because of no label on the search box. Fix it.
